agent/main_windows.go

```diff
@@ -0,0 +1,334 @@
+// +build windows
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	//	"log"
+	_ "net/http/pprof"
+	"os"
+	"path/filepath"
+	"time"
+
+	log "github.com/cihub/seelog"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/debug"
+	"golang.org/x/sys/windows/svc/eventlog"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+var elog debug.Log
+
+const ServiceName = "datadog-process-agent"
+
+// opts are the command-line options
+var defaultConfigPath = "c:\\programdata\\datadog\\datadog.yaml"
+var defaultOldConfigPath = "c:\\programdata\\datadog\\datadog.conf"
+var defaultConfdPath = "c:\\programdata\\datadog\\conf.d"
+
+var ddconfigPath string //= flag.String("ddconfig", "", "Path to the datadog.yaml configuration file")
+var ddconfdPath string  //= flag.String("ddconfd", "", "Path to the conf.d directory that contains all integration config files")
+var winopts struct {
+	installService   bool
+	uninstallService bool
+	startService     bool
+	stopService      bool
+}
```

Comment:
nit: can you wrap all of these in a single var()  block and clean up the comments with `flag` code

Reply:
Sure.  Something to think about, we use `github.com/spf13/cobra` on the metrics agent.  Might be nice to standardize.